lib/facter/patches/sysfilesystem/sys/statvfs.rb

```diff
@@ -4,6 +4,16 @@
 
 module Sys
   class Filesystem
+
+    OPT_NAMES = OPT_NAMES.dup if OPT_NAMES.frozen?
```

Comment:
Won't this issue a warning because `OPT_NAMES` is redefined?
I think you'll have to remove the constant first, then set it to the new value